### ISSUE TITLE
Implement ListFragments for Screens III-V

### ISSUE THREAD

**Issue by tooxoot** [Trikley/AwesomePlayer#3]
Implement ListFragments and transitions between them using the base Fragment from #2.
#### _Without_ touching actual media playback.

**Comment by tooxoot** [comment]
### QUESTION:
The implementation how it is now is OK for now but it appears to be **"shit-practice"**.
According to [this link](https://developer.android.com/training/basics/fragments/communicating.html) and Android-Studio's default Fragment implementation there should be some listener stuff going on! 

_I think we should go for it! What about you?_

### Solution:
See issue #6 

#### Related issues:
 #4 #2

**Comment by tooxoot** [comment]
The interface described above will be implemented in Issue #6

**Comment by tooxoot** [comment]
Done.
Implemented as:
```java
public class MediaListFragmet extends CursorListFragment{}
public class AlbumsListFragmet extends CursorListFragment{}
public class PlaylistsListFragmet extends CursorListFragment{}
```
####NOTE:
Only transitions were implemented. Deeper logic was not touched.
